README.adoc

```diff
@@ -1,36 +1,89 @@
-== Cloudbees action for building and publishing docker images using kaniko
+= CloudBees action: Build and publish Docker images with Kaniko
 
-This action can be used to build and publish Docker images using link:https://github.com/GoogleContainerTools/kaniko#kaniko---build-images-in-kubernetes[kaniko] .
+Use this action to build and publish Docker images using link:https://github.com/GoogleContainerTools/kaniko[Kaniko], which builds container images from a Dockerfile. Kaniko builds images inside a container or Kubernetes cluster.
 
-=== Prerequisites
+== Prerequisites
 
-Before invoking the action, make sure to have the docker config JSON file in the `${HOME}/.docker/config.json` path. This file is used to authenticate with the docker registry.
+To authenticate with the Docker registry, you must have a Docker config file in the `${HOME}/.docker/config.json` path.
```

Comment:
CloudBees Platform has an action to automatically generate such Docker config file from a Platform Secret.

I think  it's important to push our users to leverage our actions. Thus you can use [this](https://github.com/calculi-corp/api-gateway/blob/main/.cloudbees/workflows/test-and-deploy.yaml#L63C1-L69C59) as an example of how to use the `cloudbees-io/configure-oci-credentials`  to have Platform generate the Docker config file.